packer/images.json

```diff
@@ -84,6 +87,8 @@
       "arch": "amd64",
       "build": "dev",
       "disk_size": "50000",
+      "cpus": "3",
+      "memory": "8192",
```

Comment:
Do we have this huge amount of memory in GH Actions?  I am wondering if this could lead to swap in some way. I usually run my manual tests with 2048M of memory on qemu and I have not seen any memory issue.